biz.aQute.bndlib/src/aQute/bnd/build/BndProgress.java

```diff
@@ -0,0 +1,36 @@
+package aQute.bnd.build;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import aQute.bnd.service.progress.ProgressPlugin;
+
+public class BndProgress implements ProgressPlugin {
+
+	private final static Logger logger = LoggerFactory.getLogger(BndProgress.class);
+
+	@Override
+	public Task startTask(final String name, final int size) {
+		if (logger.isWarnEnabled()) { // warn is disabled in gradle quiet mode
```

Comment:
I guess it was not stated in my comment, but you don't need the is check any more since logger.warn will no-op if warn in not enabled.